sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetSchemaConverter.scala

```diff
@@ -356,6 +363,9 @@ private[parquet] class ParquetSchemaConverter(
       case DateType =>
         Types.primitive(INT32, repetition).as(DATE).named(field.name)
 
+      case TimestampType if writeTimestampInMillis =>
```

Comment:
We have a block of comments and codes for `TimestampType` below. Can we move this branch in the block? And we should add few comments about this change.